examples/faith/pegasus_workflow_conf.ini

```diff
@@ -0,0 +1,137 @@
+[lalapps_inspinj]
+min-mass1 = 50
+max-mass1 = 150
+min-spin1 = 0
+max-spin1 = 1
+min-mass2 = 50
+max-mass2 = 150
+min-spin2 = 0
+max-spin2 = 1
+min-mtotal = 100
+max-mtotal = 300
+enable-spin =
+aligned =
+f-lower = 20
+waveform = IMRPhenomXAS
+disable-milkyway =
+i-distr = uniform
+l-distr = random
+min-distance = 1000
+d-distr = uniform
+max-distance = 1000
+gps-start-time = 1000000000
+gps-end-time = 1000001000
+time-interval = 0.
+time-step = 1.
+seed = 123434
+m-distr = componentMass
+
+[executables]
+lalapps_inspinj = ${which:lalapps_inspinj}
+pycbc_faithsim = ${which:pycbc_faithsim}
+pycbc_faithsim_collect_results = ${which:pycbc_faithsim_collect_results}
+pycbc_faithsim_plot = ${which:pycbc_faithsim_plots}
+pycbc_splitbank = ${which:pycbc_splitbank}
+
+[workflow]
+log-path = /home/marion.pillas/projects/template_bank/log_file/
```

Comment:
Is there a way to avoid having a path to your home directory in the example?

Reply:
I can delete this path, useless